docker-compose.yaml

```diff
@@ -5,12 +5,12 @@ services:
   api:
     container_name: ${API_ADDRESS}
     build:
-      context: ../.
+      context: .
       dockerfile: docker/Dockerfile.subscription-api
     ports:
       - ${API_PORT}:${API_PORT}
     env_file:
-      - default.env
+      - .env.example
```

Comment:
So what if i want to use not example env?
I should change it at every line where it's used?

Maybe you can do it with ${ENV} so you will be able to choose env file

Reply:
I've updated project so now it is possible to pass name of env file as argument